pkg/util/utils.go

```diff
@@ -177,3 +185,47 @@ func AdjustReleaseTime(relTime time.Time, isStart bool, daysAdjustment string, c
 	}
 	return relTime
 }
+
```

Comment:
Testing helpers feel like they should be separated from the app's helpers, maybe `pkg/util/testing.go`?

Reply:
Good idea, makes the leading comment more visible too.